build_tools/python/benchmark_suites/iree/module_execution_configs.py

```diff
@@ -5,35 +5,52 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 """Defines ModuleExecutionConfig for benchmarks."""
 
+from typing import List
+
 from e2e_test_framework.definitions import iree_definitions
 from e2e_test_framework import unique_ids
 
-ELF_LOCAL_SYNC_CONFIG = iree_definitions.ModuleExecutionConfig(
+
+def _with_caching_allocator(
+    id: str,
+    tags: List[str],
+    loader: iree_definitions.RuntimeLoader,
+    driver: iree_definitions.RuntimeDriver,
+    extra_flags: List[str] = []) -> iree_definitions.ModuleExecutionConfig:
```

Comment:
Don't use mutable default values: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

```suggestion
    extra_flags: Optional[List[str]] = None) -> iree_definitions.ModuleExecutionConfig:
    extra_flags = extra_flags or []
```

Reply:
Done.